docker/packager/packager

```diff
@@ -258,6 +258,7 @@ def parse_env_variables(
         if clang_tidy:
             # 15G is not enough for tidy build
             cache_maxsize = "25G"
+            result.append("CTCACHE_DIR=/ccache/clang-tidy")
```

Comment:
What is the `CTCACHE_DIR` environment variable and where it is used?
How the `clang-tidy` program appears inside the `/ccache/` directory,
and why it is `clang-tidy`, not `clang-tidy-cache`?

Missing a comment.

Reply:
it's the directory for the cache itself. `CTCACHE_DIR` has the same purpose as the `CCACHE_DIR` above.

It's there to have the clang-tidy cache embedded into our standard `CCACHE_DIR`

Reply:
It "feels" a bit weird to embed one cache directory into the other.  Sounds safer to have both side-by-side. Also, the name "clang-tidy-cache/" would be more fitting.